doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
- 
-**Some points for CNTT to consider when contemplating recommendations:**
-   > * Despite large variances in implementation, the spectrum of networking capabilities ultimately delivered to Workloads is comparatively narrow
-   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
-   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
-   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
-   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+•	ETSI NFV does not have a separation between HW Infrastructure Management and SW Virtualisation Management and by that the cardinality of having multiple CaaS and IaaS layers on top of a shared HW Infrastructure Layer cannot be expressed
 
+•	ETSI NFV lacks a description of the reference points in between the SW Virtualisation Layers and the HW Infrastructure Layer denoted as Vl-Ha and by that cannot express Packet Flows, Control/Status Interfaces and Management Interfaces between them
 
-<a name="5.1.1"></a>
-### 5.1.1 Executive Summary
+•	ETSI NFV model does not explicitly call out an HW Abstraction layer on top of the HW resources and by that many SW Virtualisation Layers act upon the physical HW components in a non-portable way that do not allow an HW Infrastructure to compose abstracted HW resources from pools of individual HW components
+
+•	ETSI NFV lack of spelled out HW Abstraction does not enable a clear way to abstract and describe the technical possibilities that HW layer networking separation, encapsulation, acceleration, etc. can be implemented either in a HW Infrastructure Managed SmartNIC on the Server units or in the shared underlay switching fabric i.e. on the physical Switching Units 
+
+•	ETSI NFV model does not include any reference to SDN (Software Define Network) controller(s) and relevant integration points into NFVI and MANO 
+
+•	ETSI NFV model does not have a way to enable programmable forwarding planes in the HW layer controlled from higher layers of virtualization managers, orchestrators or Network Functions
+
+<p align="center"><img src="../tech/figures/ETSI.png"><b> Figure 5-1-1-1: ETSI model and Networking.</b></p>
+
+Note: Other areas of concern may be identified and addressed during the later stages of work on the RM/RA contents for Networking.
+
+<a name="5.1.1.2"></a>
+#### 5.1.1.2 Specific topics to be included in the Networking Scope
+To address the concern areas detailed above the specific topics listed below will be covered under Networking. The list below does not pretend to be complete. It will be a subject to modifications as a result of the community discussion.
+
+•	Network connectivity for L3 tenants, GWs, SDS, etc
+
+•	Spine-Leaf Architecture ( 3 Tier or 5 Tier CLOS architecture )
```

Comment:
Given CNTT is about abstraction, the key is in defining the manifestation of the connectivity at the workload reference point (i.e. what the workload sees). How the underlying fabric is bolted together, whether it's Clos or conventional, etc. are transparent to the workload. Suggest we don't try to prescribe the physical composition of the fabric.

Reply:
@markshostak I would agree hat the fabrics internal physical arrangement should be transparent for the VNF/CNF and by that should not need to be prescribed.
Then I would not know what is meant with a "conventional fabric" as being some alternative to CLOS:-)

Reply:
@TFredberg I suspect Clos would not be thrilled having his name associated with the fabric topology commonly used in cloud networks. Clos' thesis was on how to _optimize_ the network topology to require the _minimum_ connectivity necessary to achieve the desired probability of blockage. A full mesh is exactly what he was trying to **avoid**, but I digress... :-)